tests/unittests/report/util_test.py

```diff
@@ -0,0 +1,50 @@
+"""Tests for report view utility functions"""
+import sys
+import pytest
+from nav.web.report.views import find_page_range
+
+
+class TestPageRangeForPage1(object):
+    """Tests for find_page_range"""
+
+    @classmethod
+    def setup_class(cls):
+        cls.page_num = 1
+        cls.pages = range(1, 28)
+        cls.result = find_page_range(cls.page_num, cls.pages, 5)
+
+    @pytest.mark.skipif(sys.version_info >= (3, 0), reason="requires python 2")
```

Comment:
I suspect this test is unnecessary. On python 2, a sliced list makes a sliced list, and on python 3 a sliced range makes a sliced range, but the only place the return value is used, both a list and a range works (django template `for` loop).

Reply:
How about a test that tests that the return value is iterable?

Reply:
When can the below code return something that is not usable as a page range? Only if it is *passed* something that is not usable as a page range, but has a size (because `len`). If you assert that the input is suitable as a page range in the function you don't need to test that the output can serve as a page range in a unit test.

```
def find_page_range(page_number, page_range, visible_pages=5):
    """Finds a suitable page range given current page.

    Tries to make an even count of pages before and after page_number
    """
    length = len(page_range)
    page_number = int(page_number)

    if length <= visible_pages:
        return page_range

    padding = visible_pages // 2
    start = page_number - 1 - padding
    if start < 0:
        start = 0

    end = start + visible_pages
    if end >= length:
        end = length
        start = length - visible_pages

    return page_range[start:end]
```

This almost make me hunker for type annotations.